### PR TITLE
removing duplicate code in railties/lib/rails/generators/resource_helper...

### PR DESCRIPTION
**Pull request by anilmaurya**
...s.rb

### CODE REVIEW

File: railties/lib/rails/generators/resource_helpers.rb [hunk 1]
```diff
@@ -16,12 +16,10 @@ def self.included(base) #:nodoc:
       def initialize(*args) #:nodoc:
         super
         if options[:model_name]
-          controller_name = name
           self.name = options[:model_name]
```
Comment (RKushnir) [inline railties/lib/rails/generators/resource_helpers.rb:19]: Name gets changed here, so we cannot move `controller_name` assignment below this line.

Comment (iarobinson) [reply railties/lib/rails/generators/resource_helpers.rb:19]: Interesting. It appears that what looked like a cosmetic change caused unexpected downstream issues...

**Comment by Virlly** [comment]
> Looks like it is not a cosmetic change neither duplicated code. Tests are broken with this change.

**Review by RodrigoDornelles — COMMENTED** [review]
ㅤ

**Pull request closed without merging**